guides/graphite-cli/cli-command-reference.md

```diff
@@ -205,7 +219,7 @@ All `gt repo` commands can also be accessed by the shortcut `gt r`.
     * Show a rough estimate of progress through deleting branches. (This is commonly used by users running `gt repo sync` in a long-standing git repo with tens/hundreds of dead branches where Graphite has just been initialized.)
   * `-f`\
     `--force`
-    * Don't prompt confirmation when Graphite suggests to delete a branch already-merged into main or suggests to resubmit a branch whose PR base has changed.
+    * Don't prompt confirmation when Graphite suggest deleting a branch already-merged into main or suggests to resubmit a branch whose PR base has changed.
```

Comment:
Why is "already-merged" hyphenated?

Nit: "suggest" should be "suggests"

What is the default behavior here? What is the default action if no confirmation is prompted?

The wording is confusing to me here because suggestions implies "We think you should do this thing. Do you want to do it? Yes/No" and confirmation implies "You want to do this thing. Are you sure you want to do it? Yes/No"